dev/SanchiMittal/dataloader.py

```diff
@@ -0,0 +1,107 @@
+"""
+Importing dataset and pre-processing it
+"""
+import pandas as pd
+import numpy as np
+from matplotlib import pyplot as plt
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import StratifiedShuffleSplit
+import seaborn as sns
+
+# loading the data
+def load_dataset():
+    """
+    Return dataframe using pandas.
+    """
+    d = pd.read_csv("../../datasets/winequality.csv")
+    return d
+
+def visualize_data(d):
+    """
+    Visualize the dataset in terms of:
+        -distribution of target labels (recommend: True/ False).\n
+        -observing available features in the dataset.\n
+        -pairplotss of all the features with one-another.\n
+        -correlational matrix.\n
+
+    Observe patterns in the data with correlation matrix.
+
+    """
+    d["recommend"].value_counts().plot(kind="bar")
+
+    print("List of features in the dataset:")
+    features = list(d.columns.values)[:-1]
+    print(features)
+    print("\n\n")
+
+    sns.set(style="ticks", color_codes=True)
+    sns.pairplot(
+        d, vars=features, hue="recommend",
+    )
+
+    # Correelation Matrix
+    corr = d.corr()
+    print("Correlation Matrix:")
+    print(corr)
```

Comment:
I would not both printing this since you are also displaying the visualizations.